src/ted2zim/multi/scraper.py

```diff
@@ -125,7 +129,9 @@ def upload_playlists_list_to_cache(self, playlists_list, key, s3_storage):
                 s3_storage.upload_file(
                     fpath,
                     key,
-                    meta={"retrieved_on": datetime.datetime.now().isoformat()},
+                    meta={
+                        "retrieved_on": datetime.datetime.now().isoformat()  # noqa: DTZ005
```

Comment:
We should probably use utcnow* since this date is put on the S3 metadata.

*: now deprecated in 3.12, see https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated

Reply:
Yep + in the other place where we use this value to check if it is older than 7 days (otherwise we are not consistent)